From foo@baz Tue Apr  9 12:12:43 2002
Date: Wed, 21 May 2008 18:21:38 -0400
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Firmware: fix typo in example code

It's not like this stuff even builds right now, but this will make
Randy's scripts happy :)


Cc: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 samples/firmware_class/firmware_sample_firmware_class.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/samples/firmware_class/firmware_sample_firmware_class.c
+++ b/samples/firmware_class/firmware_sample_firmware_class.c
@@ -124,7 +124,7 @@ static int fw_setup_class_device(struct 
 	class_dev->class_id[BUS_ID_SIZE-1] = '\0';
 	class_dev->dev = device;
 
-	class_dev->class = &firmware_class,
+	class_dev->class = &firmware_class;
 	class_set_devdata(class_dev, fw_priv);
 	retval = class_device_register(class_dev);
 	if (retval) {
